diff --git a/node_modules/@jsr/earthstar__willow/src/store/storage/kv/kv_driver_indexeddb.js b/node_modules/@jsr/earthstar__willow/src/store/storage/kv/kv_driver_indexeddb.js
index f06844f..9b6d376 100755
--- a/node_modules/@jsr/earthstar__willow/src/store/storage/kv/kv_driver_indexeddb.js
+++ b/node_modules/@jsr/earthstar__willow/src/store/storage/kv/kv_driver_indexeddb.js
@@ -9,7 +9,7 @@ const END_LIST = Symbol("end_list");
 /** Implements {@linkcode KvDriver} on top of [IndexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API). */ export class KvDriverIndexedDB {
   db = Promise.withResolvers();
   constructor(id){
-    const request = window.indexedDB.open(`willow_${id}`, 1);
+    const request = globalThis.indexedDB.open(`willow_${id}`, 1);
     request.onerror = ()=>{
       throw new WillowError(`Could not open IndexedDB.`);
     };
diff --git a/node_modules/@jsr/earthstar__willow/src/store/storage/kv/kv_driver_indexeddb.ts b/node_modules/@jsr/earthstar__willow/src/store/storage/kv/kv_driver_indexeddb.ts
index 3ec6c81..317a7b4 100755
--- a/node_modules/@jsr/earthstar__willow/src/store/storage/kv/kv_driver_indexeddb.ts
+++ b/node_modules/@jsr/earthstar__willow/src/store/storage/kv/kv_driver_indexeddb.ts
@@ -19,7 +19,7 @@ export class KvDriverIndexedDB implements KvDriver {
   private db = Promise.withResolvers<IDBDatabase>();
 
   constructor(id: string) {
-    const request = ((window as any).indexedDB as IDBFactory).open(
+    const request = ((globalThis as any).indexedDB as IDBFactory).open(
       `willow_${id}`,
       1,
     );
diff --git a/node_modules/@jsr/earthstar__willow/src/store/storage/payload_drivers/indexeddb.js b/node_modules/@jsr/earthstar__willow/src/store/storage/payload_drivers/indexeddb.js
index dc71c4b..76ed7b5 100755
--- a/node_modules/@jsr/earthstar__willow/src/store/storage/payload_drivers/indexeddb.js
+++ b/node_modules/@jsr/earthstar__willow/src/store/storage/payload_drivers/indexeddb.js
@@ -9,7 +9,7 @@ const PARTIAL_STORE = "partial";
   constructor(id, payloadScheme){
     this.payloadScheme = payloadScheme;
     this.db = Promise.withResolvers();
-    const request = window.indexedDB.open(`willow_payloads_${id}`, 1);
+    const request = globalThis.indexedDB.open(`willow_payloads_${id}`, 1);
     request.onerror = ()=>{
       throw new WillowError(`Could not open IndexedDB.`);
     };
diff --git a/node_modules/@jsr/earthstar__willow/src/store/storage/payload_drivers/indexeddb.ts b/node_modules/@jsr/earthstar__willow/src/store/storage/payload_drivers/indexeddb.ts
index 8cbc507..0d27b89 100755
--- a/node_modules/@jsr/earthstar__willow/src/store/storage/payload_drivers/indexeddb.ts
+++ b/node_modules/@jsr/earthstar__willow/src/store/storage/payload_drivers/indexeddb.ts
@@ -16,7 +16,7 @@ export class PayloadDriverIndexedDb<PayloadDigest>
     id: string,
     readonly payloadScheme: PayloadScheme<PayloadDigest>,
   ) {
-    const request = ((window as any).indexedDB as IDBFactory).open(
+    const request = ((globalThis as any).indexedDB as IDBFactory).open(
       `willow_payloads_${id}`,
       1,
     );
